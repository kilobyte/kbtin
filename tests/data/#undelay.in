#del 1000 a
#del 1000 b
#unde {} foo
