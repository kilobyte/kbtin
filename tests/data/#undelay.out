#THAT EVENT IS NOT DEFINED.
